ServiceWebsite/ServiceWebsite.AcceptanceTests/Helpers/BookingsApiClientHelper.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Reflection;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+using RestSharp;
+using ServiceWebsite.AcceptanceTests.Configuration;
+using ServiceWebsite.AcceptanceTests.Models;
+using ServiceWebsite.AcceptanceTests.TestClients;
+using ServiceWebsite.BookingsAPI.Client;
+
+namespace ServiceWebsite.AcceptanceTests.Helpers
+{
+    public class BookingsApiClientHelper
+    {
+        public BookingsApiTestClient Client { get; set; }
+
```

Comment:
In order to add this and ensure we have the latest, we need to add an extra project to the admin-web solution (otherwise we will be editing the pipelines). If you look in the video-web solution there is a project called VideoWeb.Services.Generator, which builds the latest using nswag as part of the solution. You will need to implement that as well, otherwise any updates to the client could subsequently break the client and tests.